express/scripts/scripts.js

```diff
@@ -1531,12 +1531,6 @@ async function decorateTesting() {
   try {
     // let reason = '';
     const usp = new URLSearchParams(window.location.search);
-    const martech = usp.get('martech');
-    if ((checkTesting() && (martech !== 'off') && (martech !== 'delay')) || martech === 'rush') {
-      // eslint-disable-next-line no-console
-      console.log('rushing martech');
-      loadScript('/express/scripts/instrument.js', null, 'module');
-    }
```

Comment:
Check also maybe the return statement 8 lines lower. This would essentially bypass all instrumentation if the experiment config is invalid